Makefile

```diff
@@ -16,3 +16,4 @@ lint:
 	pylint --rcfile=.github/linters/.python-lint --fail-under=9.0 *.py
 	mypy --config-file=.github/linters/.mypy.ini *.py
 	black .
+	prettier --write README.md
```

Comment:
I can remove this line, I can see that it's giving an error on the build. Was because usually the lint was failing everytime this file was updated

Reply:
Yeah, can we please remove this line as part of this PR?  I agree we need it but the change would need to be a bit bigger.  README update re: local development, npm install of prettier in GitHub Action (linter), and this line in Makefile.